### ISSUE TITLE
would running multiple gpu speed up the process?

### ISSUE THREAD

**Issue by bitcoin-75** [brichard19/BitCrack#46]
would running multiple gpu speed up the process?

**Comment by brichard19** [comment]
Yes. Multiple GPUs can each search a different part of the keyspace.

Currently each instance of the program uses 1 GPU. You can run multiple instances,  and specify the GPU for each using the `-do option.
